.travis.yml

```diff
@@ -5,6 +5,15 @@ notifications:
   email: false
   slack: 
     andela:m0fSBLRd944nk3Gme63A9OgD
-before_script: 
-# yarn install is like npm install 
+env:
+  global:
+    - CC_TEST_REPORTER_ID=97b70dada289f07ea85b3a24ebcb83f91886f7f73c448df84071430fb75b9af6
```

Comment:
Please, please, never put tokens on Github, use Travis environment variables for them...